### PR TITLE
Re-enable psps in accceptance tests

### PR DESCRIPTION
**Pull request by ishustava**
Changes proposed in this PR:
- Allow privileged containers in the test PSP used by services deployed in tests

How I've tested this PR:
Ran GKE tests [here](https://app.circleci.com/pipelines/github/hashicorp/consul-k8s/2435/workflows/5cbda3ca-0ab4-40e0-96b0-0e725cfc76bb/jobs/14252)

How I expect reviewers to test this PR:
code review

### CODE REVIEW

File: charts/consul/test/acceptance/framework/consul/consul_cluster.go [hunk 1]
```diff
@@ -356,7 +356,7 @@ func configurePodSecurityPolicies(t *testing.T, client kubernetes.Interface, cfg
 					Name: "test-psp",
 				},
 				Spec: policyv1beta.PodSecurityPolicySpec{
-					Privileged:          false,
+					Privileged:          true,
```
Comment (ndhanushkodi) [inline charts/consul/test/acceptance/framework/consul/consul_cluster.go:359]: driveby curious: was this the fix to allow psp's to work again? Or was there another PR as well?

Comment (ndhanushkodi) [inline charts/consul/test/acceptance/framework/consul/consul_cluster.go:359]: driveby curious: was this the fix to allow psp's to work again? Or was there another PR as well?

Comment (ishustava) [reply charts/consul/test/acceptance/framework/consul/consul_cluster.go:359]: yep that's it! I think at some point between betas we've changed this setting but forgot to update the PSP we're using in tests.

**Review by lkysow — APPROVED** [review]
:thatwaseasybutton:

**Review by sadjamz — APPROVED** [review]
Thanks for catching this Iryna!
